UI/window-basic-main-outputs.cpp

```diff
@@ -698,6 +719,19 @@ void SimpleOutput::UpdateRecordingSettings_amd_cqp(int cqp)
 	obs_encoder_update(videoRecording, settings);
 }
 
+#ifdef ENABLE_HEVC
+void SimpleOutput::UpdateRecordingSettings_nvenc_hevc(int cqp)
+{
+	OBSDataAutoRelease settings = obs_data_create();
+	obs_data_set_string(settings, "rate_control", "CQP");
+	obs_data_set_string(settings, "profile", "main10");
```

Comment:
Use "main" as the default. We have code to automatically switch to "main10" if a 10-bit format is selected.

Reply:
jim-nvenc: https://github.com/obsproject/obs-studio/blob/cedc397b9f9432f5622d704c84f8a8fddef85671/plugins/obs-ffmpeg/jim-nvenc.c#L891
ffmpeg: https://github.com/FFmpeg/FFmpeg/blob/6076dbcb55d0c9b6693d1acad12a63f7268301aa/libavcodec/nvenc.c#L1274

Reply:
Done.